tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt/imported/MQTTSecurityCRLTest.java

```diff
@@ -0,0 +1,357 @@
+
+         // Subscribe to topics
+         Topic[] topics = {new Topic("test/+/some/#", QoS.AT_MOST_ONCE)};
+         connection1.subscribe(topics);
+
+         // Publish Messages
+         String payload1 = "This is message 1";
+
+         connection1.publish("test/1/some/la", payload1.getBytes(), QoS.AT_LEAST_ONCE, false);
+
+         Message message1 = connection1.receive(5, TimeUnit.SECONDS);
+
+         assertEquals(payload1, new String(message1.getPayload()));
+
+      } finally {
+         if (connection1 != null) {
+            connection1.disconnect();
+         }
+         if (server1.isStarted()) {
+            server1.stop();
+         }
+      }
+
+   }
+
+   @Test
+   public void crlNotRevokedTest() throws Exception {
+
+      ActiveMQServer server1 = initServer();
+      BlockingConnection connection1 = null;
+      try {
+         server1.start();
+
+         while (!server1.isStarted()) {
+            Thread.sleep(50);
+         }
+
+         String basedir = basedir().getPath() + "/src/test/resources/mqttCrl/client1/";
+         connection1 = retrieveMQTTConnection("ssl://localhost:1883", basedir + "truststore.jks", "changeit", basedir + "client_not_revoked.jks", "changeit");
+
+         // Subscribe to topics
+         Topic[] topics = {new Topic("test/+/some/#", QoS.AT_MOST_ONCE)};
+         connection1.subscribe(topics);
+
+         // Publish Messages
+         String payload1 = "This is message 1";
+
+         connection1.publish("test/1/some/la", payload1.getBytes(), QoS.AT_LEAST_ONCE, false);
+
+         Message message1 = connection1.receive(5, TimeUnit.SECONDS);
+
+         assertEquals(payload1, new String(message1.getPayload()));
+
+      } finally {
+         if (connection1 != null) {
+            connection1.disconnect();
+         }
+         if (server1.isStarted()) {
+            server1.stop();
+         }
+      }
+
+   }
+
+
+   private ActiveMQServer initServer() throws Exception {
+      Configuration configuration = createConfiguration("broker");
+      ActiveMQServer server = createServer(true, configuration);
+      configureBrokerSecurity(server);
+      return server;
+   }
+
+   protected Configuration createConfiguration(String name) throws Exception {
+      FileConfiguration fc = new FileConfiguration();
+
+      // we need this otherwise the data folder will be located under activemq-server and not on the temporary directory
+      fc.setPagingDirectory(getTestDir() + "/" + name + "/" + fc.getPagingDirectory());
+      fc.setLargeMessagesDirectory(getTestDir() + "/" + name + "/" + fc.getLargeMessagesDirectory());
+      fc.setJournalDirectory(getTestDir() + "/" + name + "/" + fc.getJournalDirectory());
+      fc.setBindingsDirectory(getTestDir() + "/" + name + "/" + fc.getBindingsDirectory());
+
+      addMqttTransportConfiguration(fc);
+      addWildCardConfiguration(fc);
+      fc.setSecurityEnabled(true);
```

Comment:
Is enabling security really necessary?  Isn't this test simply exercising the CRL functionality which happens at the transport layer (i.e. SSL handshake) and not at the broker's security layer?